suite-native/feature-flags/src/useIsDeviceConnectEnabled.tsx

```diff
@@ -0,0 +1,18 @@
+import { useAtom } from 'jotai';
+
+import { atomWithUnecryptedStorage } from '@suite-native/storage';
+import { isDebugEnv, isDevelopEnv } from '@suite-native/config';
+
+const isDeviceConnectEnabledAtom = atomWithUnecryptedStorage<boolean>(
+    'isDeviceConnectEnabledAtom',
+    isDevelopEnv() || isDebugEnv(),
```

Comment:
we have have a special check for both options: 
```suggestion
    isDevelopOrDebugEnv(),
```

Reply:
[c1af1be](https://github.com/trezor/trezor-suite/pull/9450/commits/c1af1be0c3c4fb06910b7dc95fdc10bcaa08b59a)